README.md

```diff
@@ -1,22 +1,195 @@
+
+#### Manage workflows
+
+**Note:** Use the `/circleci pipeline workflows` command to get the workflowID.
+
+|                             |                                     |
+| --------------------------- | ----------------------------------- |
+| `/circleci workflow get`    | Get information about a workflow.   |
+| `/circleci workflow jobs`   | Get jobs list for a given workflow. |
+| `/circleci workflow rerun`  | Rerun a given workflow.             |
+| `/circleci workflow cancel` | Cancel a given workflow.            |
+
+### Webhooks notifications
+
+Subscribe a channel to notifications from a CircleCI project.
+
+1. In the channel you want to subscribe to notifications, type `/circleci subscription add`.
+
+  -   You can add the optional flag `--only-failed` to only receive notifications about failed jobs.
+  -   You can temporarily use a project different that the one set with `/circleci default`, using the optional flag `--project <vcs/org-name/project-name>`.
+
+2. Install the [Mattermost Plugin Notify Orb for CircleCI ](https://circleci.com/developer/orbs/orb/nathanaelhoun/mattermost-plugin-notify) in your project. You usually do this by modifying the `.circleci/config.yml`.
+
+  -   You can add the command [status](https://circleci.com/developer/orbs/orb/nathanaelhoun/mattermost-plugin-notify#usage-status) in your existing jobs to get a notification when this job is finished.
+  -   Or you can set up the [approval-notification](https://circleci.com/developer/orbs/orb/nathanaelhoun/mattermost-plugin-notify#jobs-approval-notification) job in a workflow to warn that you have a workflow waiting for approval.
+
+3. Add the webhook URL given by `/circleci subscription` add to your CircleCI project.
+
+  -   You may add it to the orb as a parameter, but this is discouraged as it should be treated like a secret.
+  -   You should add it as a Environment Variable named `MM_WEBHOOK`, through the [CircleCI UI](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project) or using the plugin: `/circleci project env add MM_WEBHOOK <webhook-url>`.
+
+## Frequently Asked Questions
+
+### How does the plugin save user data for each connected CircleCI user?
+
+CircleCI user tokens are AES encrypted with an At Rest Encryption Key configured in the plugin's settings page. Once encrypted, the tokens are saved in the `PluginKeyValueStore` table in your Mattermost database.
+
+### How do I share feedback on this plugin?
+
+Wanting to share feedback on this plugin?
+
+Feel free to create a [GitHub Issue](https://github.com/mattermost/mattermost-plugin-circleci/issues) or join the [CircleCI Plugin channel](https://community.mattermost.com/core/channels/plugin-circleci) on the Mattermost Community server to discuss.
+
 ## Contributing
 
 Visit our [Development guide](https://mattermost.gitbook.io/circle-ci-plugin/contributing/development) for information about how to contribute to this plugin.
```

Comment:
```suggestion
### I saw a bug, I have a feature request or a suggestion

Please fill a [GitHub issue](https://github.com/mattermost/mattermost-plugin-circleci/issues/new/choose), it will be very useful!

## Development

Pull Requests are welcome! You can contact us on the [Mattermost Community ~Plugin: CircleCI channel](https://community.mattermost.com/core/channels/plugin-circleci).

This plugin only contains a server portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.

To avoid having to manually install your plugin, build and deploy your plugin using one of the following options.

### Deploy with Local Mode

If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. After configuring it, just run:

```
make deploy
```

### Deploy with credentials

Alternatively, you can authenticate with the server's API with a [personal access token](https://docs.mattermost.com/developer/personal-access-tokens.html):

```
export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
export MM_ADMIN_TOKEN=j44acwd8obn78cdcx7koid4jkr
make deploy
```
```